eip7594/src/lib.rs

```diff
@@ -78,12 +79,16 @@ impl Default for DASContext {
     fn default() -> Self {
         let trusted_setup = TrustedSetup::default();
         const DEFAULT_NUM_THREADS: usize = 1;
-        DASContext::with_threads(&trusted_setup, DEFAULT_NUM_THREADS)
+        DASContext::with_threads(&trusted_setup, DEFAULT_NUM_THREADS, UsePrecomp::No)
     }
 }
 
 impl DASContext {
-    pub fn with_threads(trusted_setup: &TrustedSetup, num_threads: usize) -> Self {
+    pub fn with_threads(
+        trusted_setup: &TrustedSetup,
+        num_threads: usize,
+        use_precomp: UsePrecomp,
```

Comment:
This is why its marked as breaking -- users now need to specify whether they want to use Precomp or not.

Moreover, its breaking because the Default::default method does not create the precomputations. This was the default method before, and was changed due to users probably wanting to intentionally store 100MB of data.

Reply:
Copied this into the description